### PR TITLE
Enhancement 141 admin view 01

### PR DESCRIPTION
**Pull request by Annelisebx**
_**corrected PR with limited files changed**_
*draft* 

**Resolves[ Issue 141](https://github.com/SeattleColleges/belindas-closet-android/issues/141)**
From USER STORY [109](**url**) : Admin View Page

**Completed:**
- AdminView.kt screen added for user with admin role to be routed to after login
- Admin View screen will have 3 buttons: Add Product, View All Products, and Edit User Role
- Add Product button from AdminView goes to add product page

[Library | Loom - 18 November 2023 - Watch Video](https://www.loom.com/share/c31fafba46a34c9caf393a354ecf6ef8)

image
image

**We also want to include unit tests if and when possible to make sure we're getting full coverage of our code. This will simulate a real world environment where tests are extremely important.**

[Unit Test Example](https://www.geeksforgeeks.org/unit-testing-software-testing/)

### CODE REVIEW

**Comment by bcko** [comment]
Please resolve the fix merge conflict

**Comment by Annelisebx** [comment]
Login with existing admin role now routes to a new screen called AdminView which has 3 buttons: Add Product, All Products, and Edit User Role.

File: app/src/main/java/com/example/belindas_closet/screen/Login.kt [hunk 1]
```diff
@@ -85,7 +85,7 @@ fun LoginPage(navController: NavHostController) {
                     navController.navigate(Routes.Home.route)
                 }
             ) {
-                Icon(imageVector = Icons.Default.ArrowBack, contentDescription = "Back")
+                Icon(imageVector = Icons.AutoMirrored.Filled.ArrowBack, contentDescription = "Back")
```
Comment (Robel-003) [inline app/src/main/java/com/example/belindas_closet/screen/Login.kt:88]: This import and usage of AutoMirrored is preventing the project from building and throwing an error. When I removed that import and AutoMirrored form this line, it compiled and ran.

**Review by Robel-003 — APPROVED** [review]
Aside from that, I was able to login and be routed to the admin page. Great job!

**Comment by taylorpapke** [comment]
@Annelisebx Please fix this PR so it passed CI.

**Comment by Annelisebx** [comment]
@taylorpapke  I removed the import that @Robel-003 pointed out.  I reverted it back to the previous back arrow from Main. I think CI is passing now.

**Review by taylorpapke — APPROVED** [review]
Tested this and it is working properly. I can sign in as a creator or an admin and be redirected to admin page. We will eventually need to block off creator access but we can do it in another issue.

**Pull request merged**